packages/flutter/test/material/outline_button_test.dart

```diff
@@ -9,6 +9,13 @@ import 'package:flutter/rendering.dart';
 import '../rendering/mock_canvas.dart';
 import '../widgets/semantics_tester.dart';
 
+Finder findOutlineButton() {
```

Comment:
This concerns me a bit, we won't break tests that look for OutlineButton will we?

Reply:
Good point. The tricky Finder wasn't really needed, I've removed it.